quic/s2n-quic-core/src/recovery/bbr/startup.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::recovery::{bbr, bbr::BbrCongestionController};
+use num_rational::Ratio;
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#2.6
+//# A constant specifying the minimum gain value for calculating the pacing rate that will
+//# allow the sending rate to double each round (4*ln(2) ~= 2.77)
+pub(crate) const PACING_GAIN: Ratio<u64> = Ratio::new_raw(277, 100);
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#2.6
+//# A constant specifying the minimum gain value for calculating the
+//# cwnd that will allow the sending rate to double each round (2.0)
+pub(crate) const CWND_GAIN: Ratio<u64> = Ratio::new_raw(2, 1);
+
+/// Methods related to the Startup state
+impl BbrCongestionController {
+    /// Enter the `Startup` state
+    pub fn enter_startup(&mut self) {
+        //= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.3.1.1
+        //# BBREnterStartup():
+        //#   BBR.state = Startup
+        //#   BBR.pacing_gain = BBRStartupPacingGain
+        //#   BBR.cwnd_gain = BBRStartupCwndGain
```

Comment:
I'm assuming these variables are in the state enum? Maybe make a comment about that?

Reply:
I made them methods on the state enum instead of variables, I'll add a comment